simpleclient_servlet/src/main/java/io/prometheus/client/filter/MetricsFilter.java

```diff
@@ -0,0 +1,128 @@
+package io.prometheus.client.filter;
+
+import io.prometheus.client.Histogram;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I can definitely see not wanting to pull in Apache commons. I only needed two functions from there, and I can easily write/pull in both for our use cases.

As far as location, I'm not sure where else it would go. It's a simple servlet metrics filter that has all the same dependencies as the servlet exporter filter, except that I've pulled in Mockito for testing.

Let me know if there is a better place and I can try to update that.